src/Checkout.vue

```diff
@@ -19,7 +19,11 @@ export default {
       type: String,
       required: true
     },
-    items: {
+    mode:{
+      type: String,
+      required: undefined
```

Comment:
Don't use undefined. If this prop is required, use true, otherwise, omit the `required` key.